apps/arweave/src/ar_config.hrl

```diff
@@ -44,7 +44,8 @@
 	webhooks = [],
 	max_connections = 1024,
 	max_gateway_connections = 128,
-	max_option_depth = 8
+	max_option_depth = 8,
+	active_download = false
```

Comment:
Miners need to download as many transactions as they can since it makes mining more efficient so I would make it true by default. In general, it does make sense to have it configurable because there are situations when you want to run a node which would participate in the gossip but not store data.